.github/workflows/news-check-pr.yml

```diff
@@ -47,11 +47,15 @@ jobs:
         working-directory: ./flutter/news
         run: flutter test --coverage --machine > test-results.json
 
+      - name: Remove unrequired files from coverage
+        working-directory: ./flutter/news
+        run: flutter pub global activate remove_from_coverage && flutter pub global run remove_from_coverage:remove_from_coverage -f coverage/lcov.info -r '\.g\.dart$'
```

Comment:
This is what we can use to remove autogenerated files from coverage reports

Reply:
You can create a `codecov.yml` file in your folder and add 

```ignore:
  - "**/*.g.dart"
  - "**/*.freezed.dart"
```

https://docs.codecov.com/docs/codecovyml-reference

Reply:
you are right 🤦